Documentation/building/windows-instructions.md

```diff
@@ -13,6 +13,7 @@ You must install several components to build the CoreCLR and CoreFX repos. These
 Visual Studio must be installed. Supported versions:
 - [Visual Studio 2015](https://www.visualstudio.com/vs/older-downloads/) (Community, Professional, Enterprise).  The community version is completely free.  
 - [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise).  The community version is completely free.  
+- [Visual Studio 2019 Preview](https://visualstudio.microsoft.com/vs/preview/) (Community, Professional, Enterprise).  The community version is completely free.  
 
 For Visual Studio 2015:
```

Comment:
Do we actually still support building with VS2015? I would imagine some of the newer language features might not play well.

Reply:
For a bit we supported both vs 2017 and vs 2015. I have not tried to build with it for a while, and now that arm64 builds with vs2017 it may be worth dropping the vs2015 dependency.

Reply:
In corefx we already completely dropped it.

Reply:
I'll log an issue tracking this and we can have a follow-up PR removing it if we decide to do so.

Reply:
Logged https://github.com/dotnet/coreclr/issues/22526